lib/perl/Genome/Model/Build/ReferenceSequence.pm

```diff
@@ -191,20 +191,19 @@ sub get{
 #For now, this is hard-coded
 sub get_or_create_ucsc_tiering_directory {
     my $self = shift;
-    if ($self->id eq "10194788") {
-        return "/gscmnt/sata921/info/medseq/make_tier_bed_files/hg18_build36_ucsc_files";
-    }
-    elsif ($self->id eq "106942997") {
-        return "/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-human-build37/hg19_files";
-    }
-    elsif ($self->id eq "107494762") {
-        return "/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-mouse-build37/mm9";
-    }
-    elsif ($self->id eq "3ca0ea4786dd4ebebaf3935f3d3ccab8") {
-        return "/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-mouse-build38/mm10";
-    }
-    else {
-        $self->status_message("UCSC Tiering Directory not currently available for this species: ".$self->species_name);
+
+    my %tiering_dirs = (
+        '10194788' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/hg18_build36_ucsc_files',
+        '106942997' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/hg18_build36_ucsc_files',
+        '107494762' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-mouse-build37/mm9',
+        '3ca0ea4786dd4ebebaf3935f3d3ccab8' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-mouse-build38/mm10',
+        '4ec1c5bd1f6941b8a99f2e230217cb91' => '/gscmnt/gc13001/info/model_data/jwalker_scratch/GRCh38',
```

Comment:
Would it make sense to add this to `/gscmnt/sata921/info/medseq/make_tier_bed_files/` like the others?

(It makes me a little sad these are all hard-coded, but that'd be a much bigger change.)